en/components/grids_templates/cell-editing.md

```diff
@@ -179,6 +182,9 @@ If you want to provide a custom template which will be applied when a cell is in
 > The cell template [`igxCell`](../grid/grid.md#cell-template) controls how a column's cells are shown when outside of edit mode.
 > The cell editing template directive `igxCellEditor`, handles how a column's cells in edit mode are displayed and controls the edited cell's edit value.
 
+> [!NOTE]
+> Keep in mind that by using `igxCellEditor` with a component the keyboard navigation flow will be disrupted. The same applies for a direct editing of the custom cell put in edit mode. This is because the focus will remain on the actual cell, not on the component that we've added. This is why we are taking leverage of our `igxFocus` directive, which will move the focus directly in the component and will help keeping a fluent editing flow of the cell/row.
```

Comment:
```suggestion
> By using `igxCellEditor` with any type of editor component, the keyboard navigation flow will be disrupted. The same applies to direct editing of the custom cell that enters edit mode. This is because the `focus` will remain on the `cell element`, not on the editor component that we've added - igxSelect, igxCombo and etc. This is why we should take leverage of the `igxFocus` directive, which will move the focus directly in the in-cell component and will preserve `a fluent editing flow` of the cell/row.
```